internal/bsr/container.go

```diff
@@ -28,11 +28,39 @@ const (
 	checksumFileName        = "SHA256SUM"
 	sigFileName             = "SHA256SUM.sig"
 	journalFileName         = ".journal"
+	bsrFailedCheckTemplate  = "FAILED: %s"
+)
+
+// Valid BSR checksum status values.
+const (
+	BSRCheckPass BSRChecksumStatus = "OK"
```

Comment:
I feel like this should be an enum and FAILED should be one of the BSRChecksumStatus values.